cli/common.go

```diff
@@ -35,6 +37,10 @@ var (
 
 // Init initializes the CLI environment for tectonic-torcx multicall
 func Init() error {
+	hook, err := logrus_syslog.NewSyslogHook("", "", syslog.LOG_INFO, "")
```

Comment:
I don't think I need this in the bootstrapper, output is already being logged via stdout->docker->journald. If it is feasible, can we move this to the hook only?